src/client.ts

```diff
@@ -12,6 +12,21 @@ export class Client {
    */
   public id: number;
 
+  /**
+   * Not used internally, added to allow users to assign
+   * uuids to clients if they wanted to, mainly to remove any
+   * possible type errors
+   *
+   * @example
+   * ```js
+   * server.on('connect', e => {
+   *   const client = server.clients.get(e.detail.id)
+   *   client.uuid = v4.randomUuid(); // anytime uuid is used, there will be no type errors
```

Comment:
If we want to adhere to standards, I think the [Crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) should be used.

Reply:
dont they do the same thing?

> used to generate a v4 [UUID](https://developer.mozilla.org/en-US/docs/Glossary/UUID)

Reply:
https://github.com/drashland/wocket/pull/144/commits/4062dad45a5e59c5f75d4b9f91ce099c00531682